Formula/libmongocrypt.rb

```diff
@@ -0,0 +1,33 @@
+class Libmongocrypt < Formula
+  desc "C library for Client Side Encryption"
+  homepage "https://github.com/mongodb/libmongocrypt"
+  url "https://github.com/mongodb/libmongocrypt/archive/1.0.4.tar.gz"
+  sha256 "34244b473ceedd51f5885cd80ca3f818924fac4695b5cfcd5396092c726680d9"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "mongo-c-driver" => :build
+
+  def install
+    cmake_args = std_cmake_args
+    cmake_args << if build.head?
+      "-DBUILD_VERSION=1.1.0-pre"
+    else
+      "-DBUILD_VERSION=1.0.4"
```

Comment:
Note, currently the build determines the version from the git tag. When building from a tarball without git history, the version needs to be specified manually. The C driver's formula similarly does this for HEAD builds: https://github.com/Homebrew/homebrew-core/blob/master/Formula/mongo-c-driver.rb#L25 (though it encodes the version in the tarball for releases).